scripts/package.json

```diff
@@ -16,7 +16,7 @@
     "typescript": "^4.2.3"
   },
   "dependencies": {
-    "@dcl/protocol": "^1.0.0-4799189990.commit-8494b7b",
+    "@dcl/protocol": "https://sdk-team-cdn.decentraland.org/@dcl/protocol/branch//dcl-protocol-1.0.0-4894513248.commit-bf5d705.tgz",
```

Comment:
we are ok to merge the protocol PR and use a fixed version here

Reply:
Uhm no, that will be changed when the protocol is merged, I'll update this PR to point to the latest protocol at that point, as usual.